docs/topics/wasm/wasm-configuration.md

```diff
@@ -1,20 +1,18 @@
-[//]: # (title: Troubleshooting)
+[//]: # (title: Supported versions and configuration)
```

Comment:
Could/should we cover KT-80582[1] on this page as well?


[1]: https://youtrack.jetbrains.com/issue/KT-80582/Multiple-reloads-when-using-webpack-dev-server-after-2.2.20-Beta2